### PR TITLE
Add keeper method for set/get to store

### PR DESCRIPTION
**Pull request by catShaark**
Closes: #18

### CODE REVIEW

File: x/multi-staking/types/key.go [hunk 1]
```diff
@@ -31,3 +36,28 @@ var (
 	// mem store key
 	CompletedDelegationsPrefix = []byte{0x00}
```
Comment (jiujiteiro) [inline x/multi-staking/types/key.go:37]: it looks like this key if just for the mem store, and the ones above are for the persisted store. should we still use a unique prefix? this one is the same as `BondTokenWeightKey`

File: x/multi-staking/types/key.go [hunk 2]
```diff
@@ -31,3 +36,28 @@ var (
...
+	return append(ValidatorBondDenomKey, address.MustLengthPrefix(operatorAddr)...)
+}
+
+// GetIntermediaryAccountDelegatorKey returns a key for an index containing the delegator of an intermediary account
+func GetIntermediaryAccountDelegatorKey(intermediaryAccount sdk.AccAddress) []byte {
+	return append(IntermediaryAccountDelegator, intermediaryAccount...)
+}
+
+func GetDVPairSDKBondTokensKey(delAddr sdk.AccAddress, valAddr sdk.ValAddress) []byte {
+	DVPair := append(delAddr, address.MustLengthPrefix(valAddr)...)
```
Comment (jiujiteiro) [inline x/multi-staking/types/key.go:56]: `MustLengthPrefix` has a panic scenario. Are we concerned about that here?

File: x/multi-staking/types/key.go [hunk 3]
```diff
@@ -31,3 +36,28 @@ var (
...
+	return append(DVPairSDKBondTokens, DVPair...)
+}
+
+func GetDVPairBondTokensKey(delAddr sdk.AccAddress, valAddr sdk.ValAddress) []byte {
+	DVPair := append(delAddr, address.MustLengthPrefix(valAddr)...)
```
Comment (jiujiteiro) [inline x/multi-staking/types/key.go:61]: same about `MustLengthPrefix`

File: x/multi-staking/types/key.go (hunk 2 above)
Comment (catShaark) [reply x/multi-staking/types/key.go:56]: I think it's okay to leave a panic there

File: x/multi-staking/types/key.go (hunk 1 above)
Comment (catShaark) [reply x/multi-staking/types/key.go:37]: I'll make it different just to be sure

File: x/multi-staking/types/key.go (hunk 3 above)
Comment (catShaark) [reply x/multi-staking/types/key.go:61]: I think it's okay to leave a panic there

**Comment by neitdung** [comment]
Yes, It'll be done tomorow

**Comment by neitdung** [comment]
@catShaark here is your test #19

File: x/multi-staking/types/key.go (hunk 1 above)
Comment (jiujiteiro) [reply x/multi-staking/types/key.go:37]: @catShaark will you be doing that in this pr?

**Pull request merged**